### PR TITLE
Add the ability to choose a random intro sequence each startup

### LINKED ISSUE #6284: Random intro sequence
**Describe the new feature:** Seeing as the Circles intro with its song is still in the game and can be chosen if desired, how about adding a random option in the dropdown so that both can be used at the same time and for a bit of variety? ~~Hell, maybe even add more sequences in the future, or support for custom ones?~~

**Proposal designs of the feature:** 
image

### PR DESCRIPTION
**Pull request by V1ntagezTV**
Simple "Random" intro

- Closes #6284.

### CODE REVIEW

**Comment by elleyer** [comment]
Any reason for that? Also, you should delete not English comments.

**Comment by bdach** [comment]
@elleyer The submitter aims to resolve #6284 with this.

File: osu.Game/Screens/Loader.cs [hunk 1]
```diff
@@ -58,9 +59,16 @@ protected virtual OsuScreen CreateLoadableScreen()
         }
 
         private IntroScreen getIntroSequence()
-        {
+        {//вот именно что не показывает ни всплывающих подсказок нихера
```
Comment (bdach) [inline osu.Game/Screens/Loader.cs]: Please keep all source, including comments, in English. Additionally your whitespace needs fixing (as reported by CI).

File: osu.Game/Screens/Loader.cs [hunk 2]
```diff
@@ -58,9 +59,16 @@ protected virtual OsuScreen CreateLoadableScreen()
...
+            Random random = new Random();
```
Comment (bdach) [inline osu.Game/Screens/Loader.cs]: * Use `var` here.
* This allocation can be moved into the `switch` case.

**Review by bdach — CHANGES_REQUESTED** [review]

**Review by frenzibyte — CHANGES_REQUESTED** [review]
This can be fairly simplified by just picking a random intro sequence enum value.

```csharp
switch (introSequence)
{
    case IntroSequence.Random:
        var introSequences = Enum.GetValues(typeof(IntroSequence)).Except(new[] { IntroSequence.Random }).ToList();
        return getIntroSequence(introSequences[RNG.Next(introSequences.Count)]);
}
```

Notes to be took from the above snippet:
 - `RNG` is used instead of `Random` which is a simplified way of seedless randomizing.

**Review by peppy — APPROVED** [review]

**Comment by smileyhead** [comment]
Nice, thanks! :-)

**Pull request merged**